### PR TITLE
Overview page acceptance tests

### PR DESCRIPTION
**Pull request by adlius**
- Ticket: []
- Feature flag: n/a

### CODE REVIEW

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+            stripHtmlTags(t('registries.overview.pending.long_description')),
+            'Long description matches the reviewsState of the registration',
+        );
+        await click('[data-test-moderation-dropdown-button]');
+        assert.dom('[data-test-moderation-dropdown-decision-label]').exists({ count: 2 }, 'Two options are available');
+        assert.dom('[data-test-moderation-dropdown-decision-label="accept_submission"]').exists(
+            'Option to accept submission exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="accept_submission"]').hasText(
+            'Accept submission',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:57]: would these assertions be more robust if we checked against `t(registries.makeDecisionDropdown.acceptSubmission)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+            'Accept submission option has correct text',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="reject_submission"]').exists(
+            'Option to reject submission exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="reject_submission"]').hasText(
+            'Reject submission',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:64]: ... and checked against `t(registries.makeDecisionDropdown.rejectSubmission)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+        await click('[data-test-moderation-dropdown-button]');
+        assert.dom('[data-test-moderation-dropdown-decision-label]').exists(
+            { count: 1 },
+            'Only one option is available',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="force_withdraw"]').exists(
+            'Option to force withdraw exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="force_withdraw"]').hasText(
+            'Force withdraw',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:131]: `t(registries.makeDecisionDropdown.forceWithdraw)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+            stripHtmlTags(t('registries.overview.pendingWithdraw.long_description')),
+            'Long description matches the reviewsState of the registration',
+        );
+        await click('[data-test-moderation-dropdown-button]');
+        assert.dom('[data-test-moderation-dropdown-decision-label]').exists({ count: 2 }, 'Two options are available');
+        assert.dom('[data-test-moderation-dropdown-decision-label="accept_withdrawal"]').exists(
+            'Option to accept withdraw exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="accept_withdrawal"]').hasText(
+            'Accept withdrawal',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:165]: `t(registries.makeDecisionDropdown.acceptWithdrawal)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+            'Accept withdrawal option has correct text',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="reject_withdrawal"]').exists(
+            'Option to reject withdraw exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="reject_withdrawal"]').hasText(
+            'Reject withdrawal',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:172]: `t(registries.makeDecisionDropdown.rejectWithdrawal)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+        await click('[data-test-moderation-dropdown-button]');
+        assert.dom('[data-test-moderation-dropdown-decision-label]').exists(
+            { count: 1 },
+            'Only one option is available',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="force_withdraw"]').exists(
+            'Option to force withdraw exists',
+        );
+        assert.dom('[data-test-moderation-dropdown-decision-label="force_withdraw"]').hasText(
+            'Force withdraw',
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:234]: `t(registries.makeDecisionDropdown.forceWithdraw)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 7]
```diff
@@ -0,0 +1,325 @@
...
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:276]: `t(registries.makeDecisionDropdown.forceWithdraw)`?

File: tests/engines/registries/acceptance/overview/moderator-mode-test.ts [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
```
Comment (futa-ikeda) [inline tests/engines/registries/acceptance/overview/moderator-mode-test.ts:317]: `t(registries.makeDecisionDropdown.forceWithdraw)`?

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Looks great! Small suggestion with regards to the decision labels checking against a translated string, as opposed to a raw string.

**Comment by adlius** [comment]
> Looks great! Small suggestion with regards to the decision labels checking against a translated string, as opposed to a raw string.

The only problem is that the casing is different. In the template we use `humanize` instead of the translation strings.

**Comment by futa-ikeda** [comment]
Ahh, I see now. I guess this goes back to the question of if we want to use sanitized keys/values passed from the backend as translations, but I think we can address that some other time

**Review by futa-ikeda — APPROVED** [review]

**Review by fabmiz — APPROVED** [review]
💯

**Pull request merged**